src/main/java/de/javakaffee/kryoserializers/guava/ImmutableMapSerializer.java

```diff
@@ -54,5 +54,17 @@ public static void registerSerializers(final Kryo kryo) {
         kryo.register(ImmutableMap.of(o1, o1).getClass(), serializer);
         kryo.register(ImmutableMap.of(o1, o1, o2, o2).getClass(), serializer);
 
+        Map<DummyEnum,Object> enumMap = new EnumMap<DummyEnum, Object>(DummyEnum.class);
+        for (DummyEnum e : DummyEnum.values()) {
+            enumMap.put(e, o1);
```

Comment:
Yes, it's not the best, if the Guava team decides to change the internal algorithm it might break this, but we don't have any other solutions to access this class.